src/modules/components/member/memberDataListItem/memberDataListItem.test.tsx

```diff
@@ -0,0 +1,135 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import { getAddress, isAddress } from 'viem';
+import { useAccount, useEnsAddress, useEnsAvatar, useEnsName } from 'wagmi';
+import { DataList } from '../../../../core';
+import { MemberDataListItemStructure, type IMemberDataListItemProps } from './memberDataListItem';
+
+jest.mock('viem', () => ({
+    isAddress: jest.fn(),
+    getAddress: jest.fn(),
+}));
+
+jest.mock('viem/ens', () => ({
+    normalize: jest.fn(),
+}));
+
+jest.mock('wagmi', () => ({
+    useEnsAddress: jest.fn(),
+    useEnsName: jest.fn(),
+    useEnsAvatar: jest.fn(),
+    useAccount: jest.fn(),
+}));
+
+describe('<MemberDataListItem /> component', () => {
+    const createTestComponent = (props?: Partial<IMemberDataListItemProps>) => {
+        const completeProps: IMemberDataListItemProps = {
+            ...props,
+        };
+
+        return (
+            <DataList.Root entityLabel="Members">
+                <DataList.Container>
+                    <MemberDataListItemStructure {...completeProps} />
+                </DataList.Container>{' '}
+            </DataList.Root>
+        );
+    };
+
+    const originalGlobalImage = global.Image;
+
+    beforeAll(() => {
+        (window.Image as unknown) = class MockImage {
+            onload: () => void = () => {};
+            src: string = '';
+            constructor() {
+                setTimeout(() => {
+                    this.onload();
+                }, 100);
+            }
+        };
+    });
+
+    afterAll(() => {
+        global.Image = originalGlobalImage;
+    });
```

Comment:
Let's simplify the tests here by simply mocking the `MemberAvatar` component. In that way we don't need to know how the avatar/member-avatar components work, we can remove the window.image mock and the useEnsAvatar mocks as well. You can mock at the beginning of the file like:
```
jest.mock('../memberAvatar', () => ({ MemberAvatar: () => <div data-testid="member-avatar-mock" /> }));
```